Python/Product/PythonTools/PythonTools/Debugger/DTEDebuggerExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.PythonTools.Debugger.DebugEngine;
+using Microsoft.VisualStudio.Debugger.Interop;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.PythonTools.Debugger
+{
```

Comment:
We do opening braces on same line for everything. Otherwise, default C# formatting settings.